src/client/serialize/SConscript

```diff
@@ -0,0 +1,31 @@
+"""Build FUSE client"""
+import daos_build
+
+def scons():
```

Comment:
(pylint-inconsistent-return-statements) Either all return statements in a function should return an expression, or none of them should.